### ISSUE TITLE
Not working with UCD Deploy version 7.2.2.1

### ISSUE THREAD

**Issue by ameritasmb** [UrbanCode/Send-SMTP-Email-UCD#7]
We recently updated to UCD 7.2.21 from 7.0.2.3ifx01 and receive the following error when  executing the Send Mail step:

org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/opt/ibm-ucd/agent/var/plugins/com.urbancode.air.plugin.SendEmail_1_a1de81c6b0f9870b3d7e7e649a1ad3d3403009f5b8b5b792abb7136a05930e48/classes/com/urbancode/air/AirPluginTool.groovy: 34: The variable [inputPropsStream] is declared final but is reassigned
. At [34:13]  @ line 34, column 13.
               inputPropsStream = new FileInputStream(inputPropsFile);
               ^
 
1 error

**Comment by reachsenthilnathan** [comment]
It might be due to groovy upgrade. *my guess* I have asked a developer to look at this. 

- Senthil Nathan - UCD Product Manager

**Comment by ameritasmb** [comment]
I agree.  There were a few other IBM supported plugins that we saw had similar errors, but were fine after we updated to the latest versions.

**Comment by CLSuerstedt** [comment]
We are having the same issue with this plugin.  I have a ticket open with support, but we don't seem to be getting any traction.  This is a legacy plugin that IBM originally helped us configure when we used services to install UCD many years ago.
